middlewares/schema_validator.js

```diff
@@ -0,0 +1,69 @@
+const Joi = require('joi');
+const _ = require('lodash');
+const Schemas = require('../models/schemas');
+
+module.exports = (useJoiError = false) => {
+    const _useJoiError = _.isBoolean(useJoiError) && useJoiError;
+
+    const _supportedMethods  = ['post', 'put'];
+
+    const _validationOptions = {
+        abortEarly: false, // abort after the last validation error
+        allowUnknown: true, // allow unknown keys that will be ignored
+        stripUnknown: true // remove unknown keys from the validated data
+    }
+
+    return (req, res, next) => {
+
+        const route = req.route.path;
+        const method = req.method.toLowerCase();
+
+        if (_.includes(_supportedMethods, method) && _.has(Schemas, route)) {
```

Comment:
Are you checking to see if there is a validation schema for the route and if it is using one of he supported methods